macro/src/expand.rs

```diff
@@ -667,8 +667,12 @@ fn expand_type_alias(alias: &TypeAlias) -> TokenStream {
 }
 
 fn expand_type_alias_verify(namespace: &Namespace, alias: &TypeAlias) -> TokenStream {
+    let namespace = alias.namespace.as_ref().unwrap_or(namespace);
+    // Review TODO: Is this unwrap fine? i.e. is it ok to assume that, if the
+    // TypePath parsed, that it has at least one segment?
```

Comment:
Review question for you dtolnay@ (and there's a similar unwrap() in write.rs above)